src/webgl/material.js

```diff
@@ -178,7 +178,7 @@ p5.prototype.ambientMaterial = function(v1, v2, v3, a) {
   gl.useProgram(shaderProgram);
   shaderProgram.uMaterialColor = gl.getUniformLocation(
     shaderProgram, 'uMaterialColor' );
-  var colors = this._renderer._applyColorBlend(v1,v2,v3,a);
+  var colors = this._renderer._applyColorBlend.apply(this._renderer, arguments);
```

Comment:
@karenpeng Sorry for the delay getting back to you on this.  Yes, good point.  Could you pull this bug fix into a separate PR and please submit?  I think the camera changes are a larger conversation, but you're 100% right about the _applyColorBlend bug and I'd like to get this patched ASAP.  Thanks!